decidim-accountability/app/controllers/decidim/accountability/application_controller.rb

```diff
@@ -8,7 +8,6 @@ module Accountability
     # Note that it inherits from `Decidim::Components::BaseController`, which
     # override its layout and provide all kinds of useful methods.
     class ApplicationController < Decidim::Components::BaseController
```

Comment:
I have issues understanding this change. 

Simplified this method is setting a participatory space layout, and sets some permissions using authorize_participatory_space before action ... 

In some places you are using `participatory_space_layout` method that is setting the participatory space layout and sets some permissions using authorize_participatory_space before action ... 

By removing this method, you would relay on setting the permission using the before action from `Decidim::Components::BaseController`, without rendering any layout. 


My questions would be: 
- Why aren't you using `participatory_space_layout` method in the BaseController:  to properly set the layout and permission 
- Why is there the inconsistency ?

Reply:
About this point I think that the participatory_space_layout may be removed. From what I seemed to understand in the Decidim prior to redesign, this method was used to use the participatory space layout defined in its manifest and authorize access to the participatory space in the involved actions. This was useful for components to be displayed in the proper space layout. With the redesign the layout is simplified and the components are shown with the general application layout. From the components it's still required to authorize access to the participatory space but the layout part is no longer required. On the other hand, the layouts of each space are simplified a lot and consist fundamentally on the application layout including the specific assets of each one.

I think this part could be simplified, but the task could be extracted to an issue to not block the merge. Also I would prefer to do this after removing any traces of redesign/legacy design including assets renaming.

I have removed the `participatory_space_layout` from the elections component because is not necessary

Reply:
This has been extracted to https://github.com/decidim/decidim/issues/11410